packages/web/components/cards/hero-card.tsx

```diff
@@ -95,6 +95,13 @@ export const HeroCard: React.FunctionComponent<{
           .heroImage:hover .backgroundImage {
             transform: scale(1.15);
           }
+          .gradient {
```

Comment:
It didn't work correctly with the gradient in tailwind. If anyone knows the correct tailwind classes I'm all ears.

Reply:
They're added as extensions to the config:

https://github.com/osmosis-labs/osmosis-frontend/blob/7ffa25d8ff0bd2c738a9a75914e69493392529a0/packages/web/tailwind.config.js#L167-L175